Prog/SF_2012_16/Acs_sf_2012_16_dc.sas

```diff
@@ -24,7 +24,10 @@
   state_ab = DC,
 
   /** Year range (xxxx_yy). Ex: 2005_09 **/
-  years = 2012_16
+  years = 2012_16,
+  
+  /** Description of latest file revisions for metadata **/
+  revisions = %str(New file.)
```

Comment:
This wasn't previously in the instructions, but since these are existing files that we are modifying, we should include an updated _revisions=_ note for the metadata system that documents the change. In this case, replace `revisions = %str(New file.)` with

`revisions = %str(Added tables B25031 B25041 B25042 B25068 B05006 B27001 B25095 B25074 B14001.)`

Do the same for the MD, VA, and WV programs.